From 17c87ac58ce6d475fa1987347fcb868ef7f75e40 Mon Sep 17 00:00:00 2001
From: Joseph LaFreniere <joseph@lafreniere.xyz>
Date: Sun, 2 Jun 2019 12:38:27 -0500
Subject: [PATCH] Replace executable domainname with hostname

Fixes build failure:
> ice-9/boot-9.scm:752:25: In procedure dispatch-exception:
> patch-el-files: unable to locate  "domainname"
---
 hypb.el | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/hypb.el b/hypb.el
index 6abc17a..0332946 100644
--- a/hypb.el
+++ b/hypb.el
@@ -183,8 +183,7 @@ Global keymap is used unless optional KEYMAP is given."
 
 (defun hypb:domain-name ()
   "Returns current Internet domain name with '@' prepended or nil if none."
-  (let* ((dname-cmd (or (file-exists-p "/usr/bin/domainname")
-			(file-exists-p "/bin/domainname")))
+  (let* ((dname-cmd (file-exists-p "/bin/hostname"))
 	 (dname (or (and (boundp 'message-user-fqdn) (stringp message-user-fqdn)
 			 (string-match "\\." message-user-fqdn)
 			 message-user-fqdn)
-- 
2.21.0

